src/main/java/com/minecolonies/entity/pathfinding/PathNavigate.java

```diff
@@ -263,6 +279,35 @@ else if (entity.isInWater())
         }
     }
 
+    @Override
+    protected void pathFollow()
+    {
+        int curNode = currentPath.getCurrentPathIndex();
+        int curNodeNext = curNode + 1;
+        if (curNodeNext < currentPath.getCurrentPathLength())
+        {
+            PathPointExtended pEx = (PathPointExtended) currentPath.getPathPointFromIndex(curNode);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'pEx' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)